notebooks/dataframes/dataframe.ipynb

```diff
@@ -1,5 +1,27 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```

Comment:
Just curious, how much do we care about these `execution_count` in notebooks? We are having two execution_count = 1 in this file.

Reply:
I don't think we really care. I recommend using the ReviewNB link above to review as a rendered version: https://github.com/googleapis/python-bigquery-dataframes/pull/519#issuecomment-2021335773